source/system-administrators/deployer/templates-guide.rst

```diff
@@ -0,0 +1,244 @@
++------------------------------+-----------+------------------------------------------------------------------------+
+|``aws.s3.url``                ||checkmark||The full AWS S3 URI of the folder to sync files                         |
++------------------------------+-----------+------------------------------------------------------------------------+
+|``disable_deploy_cron``       |           |Disables the cron job that runs deployments every certain amount of time|
++------------------------------+-----------+------------------------------------------------------------------------+
+|``local_repo_path``           |           |The path of the local Git repository to use                             |
++------------------------------+-----------+------------------------------------------------------------------------+
+|``repo_branch``               |           |The branch name of the remote Git repo to pull from                     |
++------------------------------+-----------+------------------------------------------------------------------------+
+|``repo_username``             |           |Username to access remote repository                                    |
++------------------------------+-----------+------------------------------------------------------------------------+
+|``repo_password``             |           |Password to access remote repository                                    |
++------------------------------+-----------+------------------------------------------------------------------------+
+|``ssh_private_key_path``      |           |The path for the private key to access remote repository                |
++------------------------------+-----------+------------------------------------------------------------------------+
+|``ssh_private_key_passphrase``|           |The passphrase for the private key to access remote repository (only if |
+|                              |           |the key is passphrase-protected)                                        |
++------------------------------+-----------+------------------------------------------------------------------------+
+|``notification_addresses``    |           |The email addresses that should receive deployment notifications        |
++------------------------------+-----------+------------------------------------------------------------------------+
+
+.. note:: When this target is used, the value of ``repo_url`` must be a supported Git URL (HTTP or SSH)
+
+.. note:: For more details about setting up a serverless delivery see :ref:`setup-serverless-site`
+
+^^^^^^^^^^^^^^^^^^^^^^
+AWS Cloudformed Target
+^^^^^^^^^^^^^^^^^^^^^^
+
+This template is used to provide a serverless delivery environment without the need to manually create all required 
+resources in AWS. It works similar to the AWS S3 Target but uses an AWS Cloudformation template.
+
```

Comment:
Change to: `...but uses an AWS CloudFormation template to create the AWS resources on target creation: the S3 bucket where the site content will be stored and a CloudFront Distribution that will deliver the static assets directly from the S3 bucket. These resources will be deleted when the target is deleted`.

Make sure whenever you mention `CloudFormation` it is with a capital F.